### PR TITLE
Fixed HoloLensCaptureExporter handling of NV12-encoded video/preview streams

### PR DESCRIPTION
**Pull request by AshleyF**

### CODE REVIEW

File: Sources/MixedReality/HoloLensCapture/HoloLensCaptureExporter/DataExporter.cs [hunk 1]
```diff
@@ -116,21 +117,29 @@ void VerifyMutualExclusivity(dynamic s0, dynamic s1)
 
                 if (encodedImageCameraView != null)
                 {
-                    // export encoded camera view as is
                     VerifyMutualExclusivity(imageCameraView, gzipImageCameraView);
-                    encodedImageCameraView.Export(name, exportCommand.OutputPath, streamWritersToClose);
+                    if (isNV12)
+                    {
+                        // export NV12-encoded camera view as lossless PNG
+                        encodedImageCameraView?.Decode(decoder)?.Encode(pngEncoder).Export(name, exportCommand.OutputPath, streamWritersToClose);
```
Comment (danbohus) [inline Sources/MixedReality/HoloLensCapture/HoloLensCaptureExporter/DataExporter.cs:124]: Delivery policies? We need to make sure we're not dropping messages.

Comment (AshleyF) [reply Sources/MixedReality/HoloLensCapture/HoloLensCaptureExporter/DataExporter.cs:124]: The whole pipeline is created with `DeliveryPolicy.SynchronousOrThrottle`.

Comment (sandrist) [reply Sources/MixedReality/HoloLensCapture/HoloLensCaptureExporter/DataExporter.cs:124]: I think we want `DeliveryPolicy.Unlimited` on the pipeline, right? `SynchronousOrThrottle` might sometimes drop messages if the synchronous attempt fails and the component cannot be throttled? Or maybe not, but still, seems more straightforward to just specify `Unlimited`.

**Review by sandrist — APPROVED** [review]
:shipit:

**Pull request merged**